docs/entrance randomization.md

```diff
@@ -72,6 +75,11 @@ logic to generate a valid world layout by connecting the partially connected edg
 that, your region graph might look something like the following diagram. Note how each randomizable entrance/exit pair
 (represented as a bidirectional arrow) is disconnected on one end.
 
+> [!WARNING]
+> It is highly recommended to use explicit indirect conditions when using Generic ER. Failure to do so may lead to an
+> increased failure rate if your dead end regions are required in logic rules as Generic ER would have no way to
+> determine they are required when performing randomization and treat them accordingly.
```

Comment:
Wondering if this should straight up error, haha

Reply:
I guess there may be worlds that are super open and have like, a single indirect condition in a non dead end region. And it never leads to problems. ~~But also a world like that should be using explicit indirect conditions~~

Reply:
I think you are probably right, I can make this error. The better option imo is to make this not a warning ie to solve for implicit indirect conditions but I'm pretty sure that is not possible without some change to rules that allows better static analysis

Reply:
It's kind of possible to detect if you monkeypatch the hell out of core, but yeah, I wouldn't recommend it